lib/asar.js

```diff
@@ -1,5 +1,5 @@
 'use strict'
-const fs = require('fs')
+const fs = process.versions.electron ? require('original-fs') : require('fs')
```

Comment:
This is repeated in a number of places, is it worth moving this condition and require in to some kind of proxy module that returns either `original-fs` or `fs` depending on the environment?